docs/modules/ROOT/pages/adr/0042-backup-encryption-for-mariadb-galera-and-postgresql.adoc

```diff
@@ -0,0 +1,167 @@
+= ADR 0042 - Backup Encryption for MariaDB Galera and PostgreSQL
+:adr_author:    Mike Ditton
+:adr_owner:     Schedar
+:adr_reviewers: Schedar
+:adr_date:      2025-10-06
+:adr_upd_date:  2025-10-06
+:adr_status:    draft
+:adr_tags:      backup,encryption,galera,postgresql
+
+include::partial$adr-meta.adoc[]
+
+[NOTE]
+.Summary
+====
+Use rclone as an S3 encryption proxy to provide encrypted backups for MariaDB Galera and PostgreSQL services.
+====
+
+== Context
+
+We want to provide encrypted backups for all AppCat services. The backup solutions for MariaDB and PostgreSQL are tightly integrated with their respective operators, which prevents us from using k8up. Neither the MariaDB Operator nor the PostgreSQL Operator support encrypted backups natively, so we need an alternative solution.
+
+== Evaluated Options
+
+=== Constellation s3proxy - not viable
+
+The s3proxy is part of the Constellation Kubernetes engine and provides transparent client-side encryption for S3-compatible storage backends. This would enable seamless integration with PostgreSQL and Galera.
```

Comment:
Please add links to the mentioned tools for proper context, e.g.

`https://s3proxy.com[s3proxy^]`